osu.Framework/Platform/SDL2DesktopWindow.cs

```diff
@@ -1263,6 +1264,8 @@ private static DisplayMode displayModeFromSDL(SDL.SDL_DisplayMode mode, int disp
             return new DisplayMode(SDL.SDL_GetPixelFormatName(mode.format), new Size(mode.w, mode.h), bpp, mode.refresh_rate, modeIndex, displayIndex);
         }
 
+        private static Size scaleSize(Size size, float scale) => new Size((int)(size.Width * scale), (int)(size.Height * scale));
```

Comment:
can be written as

```suggestion
        private static Size scaleSize(Size size, float scale) => Size.Round(size * scale);
```

or

```suggestion
        private static Size scaleSize(Size size, float scale) => (size * scale).ToSize(); // truncates
```

at which point the helper is somewhat questionable

Reply:
Oh, I didn't realize you could do that..., will remove.